examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -5,12 +5,14 @@ using Trixi
 # semidiscretization of the ideal compressible Navier-Stokes equations
 
 prandtl_number() = 0.72
-mu() = 0.01
+@inline function mu(u, equations)
+    return 0.01
+end
```

Comment:
```suggestion
mu(u, equations) = 0.01
```
Why not like this? Less visual clutter IMHO, but also a matter of personal preference, so 🤷‍♂️

Also, I am relatively sure that the `@inline` is not needed - could you verify? If not needed, it makes it even easier to read

Reply:
Also applies to all the other pertinent elixirs

Reply:
Makes sense :+1: